_includes/tweet_this/index.html

```diff
@@ -0,0 +1,4 @@
+<div class="pa3">
+  <div class="f3 sans-serif i">{{ include.content }}</div>
+  <a class="v-mid blue link pt3 dib" href="https://twitter.com/share?url={{ page.url | absolute_url }}&text={{ include.content }}" target="_blank"><span class="w2 dib v-mid">{% include tweet_this/twitter.svg %}</span><span class="v-mid pl2 f5">Tweet this</span></a>
```

Comment:
I suggest you add `&via=learnk8s` to the URL 😉

Reply:
good idea